internal/bminventory/inventory.go

```diff
@@ -5118,9 +5100,8 @@ func (b *bareMetalInventory) v1uploadLogs(ctx context.Context, params installer.
 			return err
 		}
 
-		b.eventsHandler.AddEvent(ctx, params.ClusterID, params.HostID, models.EventSeverityInfo,
-			fmt.Sprintf("Uploaded logs for host %s cluster %s",
-				hostutil.GetHostnameForMsg(&dbHost.Host), params.ClusterID.String()), time.Now())
+		eventgen.SendHostLogsUploadedEvent(ctx, b.eventsHandler, &params.ClusterID, *params.HostID, params.ClusterID,
+			hostutil.GetHostnameForMsg(&dbHost.Host))
```

Comment:
the fifth argument should be infra_env_id

Reply:
As mentioned here https://github.com/openshift/assisted-service/pull/2717#discussion_r727761899, since it's a V1 function, we assume here that `infra-env-id = cluster-id`